src/core_ocean/shared/mpas_ocn_vmix_gotm.F

```diff
@@ -0,0 +1,531 @@
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  ocn_vmix_gotm
+!
+!> \brief MPAS ocean vertical mixing interface to GOTM
+!> \author Qing Li
+!> \date   11 May 2020
+!> \details
+!>  This module contains the routines for calls into GOTM
+!>
+!
+!-----------------------------------------------------------------------
+
+module ocn_vmix_gotm
+
+   use mpas_derived_types
+   use mpas_pool_routines
+   use mpas_timer
+   use mpas_constants
+   use mpas_log
+   use mpas_timekeeping
+
+   use ocn_constants
+   use ocn_config
+   use ocn_diagnostics_variables
+
+   ! TODO: read in GOTM namelist for testing <20200512, Qing Li> !
+   use mpas_io_units
```

Comment:
@qingli411 is there something to do here, or should this comment be removed?

Reply:
@mark-petersen I think this comment can be removed. I will check and clean up all the lines with TODO.